styles/navigation.css

```diff
@@ -0,0 +1,60 @@
+.navbarMain_body {
+    flex-wrap: wrap;
+    gap: 3rem;
+}
+
+.ecom_header {
+    width: 100%;
+    background-color: var(--light-grey);
+    justify-content: space-between;
+    padding: 1rem;
+}
+
+.nav {
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+}
+
+.cart_btn {
+    background-color: var(--light-grey);
+}
+
+.brand_name a {
+    color: var(--primary-color);
+}
+
+.large_icon {
+    color: var(--primary-color);
+    font-size: 1.5rem;
+}
+
+.input_Serch_container {
+    flex-basis: 30%;
+}
+
+.input_serch {
+    width: 100%;
+    border-radius: 5px;
+    padding: 10px;
```

Comment:
i think 1rem would work here

Reply:
yes thank you i will modify in next commit